twyla/service/events.py

```diff
@@ -1,47 +1,22 @@
 from aioamqp.protocol import OPEN
 
-import twyla.service.queues as queues
+from twyla.service import queues
 
+class EventBus:
 
-async def listen(event_name):
-    """
-    listen is a small package level wrapper around QueueManager.listen for ease
-    of use.
+    def __init__(self, config_prefix, group, emits):
```

Comment:
Should we add type annotations? I know that we haven't used them as much elsewhere in the package but it would be nice to slowly introduce them since it makes it easier to reason on-the-fly.

Reply:
that makes sense